src/main/java/org/spongepowered/api/text/channel/MessageChannel.java

```diff
@@ -134,15 +137,45 @@ default void send(Text original) {
     }
 
     /**
+     * Sends a message to this channel, transforming and sending it to
+     * the members.
+     *
+     * <p>By default, the {@link ChatTypes#SYSTEM} type is used when sending
+     * messages to applicable recipients of this channel.</p>
+     *
+     * @param original The original text to send
+     * @param type The type of message
+     */
+    default void send(Text original, ChatType type) {
```

Comment:
`send` methods should be able to control it - a message channel is a way to distribute and transform messages in bulk, there's no reason at all this should not be supported.